doc/source/configuration/shared-file-systems/drivers/vastdata_driver.rst

```diff
@@ -45,11 +47,12 @@ share driver.
 .. include:: ../../tables/manila-vastdata.inc
 
 
-Vastdata driver configuration example
+VASTData driver configuration example
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The following parameters shows a sample subset of the ``manila.conf`` file,
```

Comment:
Suggest changing the whole paragraph to something like this:

The following example shows parameters in the manila.conf file that are used to configure VAST Share Driver. They include two options under [DEFAULT] and parameters under [vast]. 
Note that a real manila.conf file would also include other parameters that are not specific to VAST Share Driver.